### DISCUSSION TITLE
Could not find com.github.pedroSG94.rtmp-rtsp-stream-client-java:rtplibrary:2.2.0

### DISCUSSION THREAD

**Discussion by AlexUrrutia** [pedroSG94/RootEncoder#1183] [General]
<img width="649" alt="Screen Shot 2022-10-28 at 6 43 35 AM" src="https://user-images.githubusercontent.com/7042361/198579507-9c7b3f5a-18bc-4bd6-8d1a-65293b3853d9.png">
Hello @pedroSG94 ,
can't find the latest version

**Comment by pedroSG94** [comment]
Hello,

Updated to 2.2.1. The error should be solved now
